elasticsearch_collectd.py

```diff
@@ -628,10 +643,28 @@ def fetch_stats():
                 parse_index_stats(indexes_json_stats[index_name], index_name)
 
 
+# SSL requests wrapper
+class HTTPSClientAuthHandler(urllib2.HTTPSHandler):
+    def __init__(self, key_file, cert_file):
+        urllib2.HTTPSHandler.__init__(self)
+        self.key_file = key_file
+        self.cert_file = cert_file
+
+    def https_open(self, req):
+        return self.do_open(self.get_connection, req)
+
+    def get_connection(self, host, timeout=300):
+        return httplib.HTTPSConnection(host, key_file=self.key_file, cert_file=self.cert_file, timeout=timeout)
+
+
 def fetch_url(url):
     response = None
     try:
-        response = urllib2.urlopen(url, timeout=10)
+        if ES_HTTP_TLS_ENABLED:
+            opener = urllib2.build_opener(HTTPSClientAuthHandler(ES_TLS_KEY_PATH, ES_TLS_CERT_PATH))
```

Comment:
super minor nit: is the change in timeout here intentional/significant?

Reply:
No that should be the same, good catch.